Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -45,11 +45,12 @@ class CredentialsManagerSpec: QuickSpec {
     override func spec() {
 
         let authentication = Auth0.authentication(clientId: ClientId, domain: Domain)
+        let storage = A0SimpleKeychain(service: "test_service")
         var credentialsManager: CredentialsManager!
         var credentials: Credentials!
 
         beforeEach {
-            credentialsManager = CredentialsManager(authentication: authentication)
+            credentialsManager = CredentialsManager(authentication: authentication, storage: storage)
```

Comment:
I don't think you should change the default behaviour to use a custom storage for tests, this is your use case but it is not the 90%+ customer use case.
It makes sense now that you can change the keychain to add a test here for the `credentials` service as this is default.